app/assets/javascripts/miq_application.js

```diff
@@ -1484,6 +1484,10 @@ function miqToolbarOnClick(_e) {
       } else {
         params = miqSerializeForm(button.data('url_parms'));
       }
+    } else if (button.data('url_parms').match("id=LIST")) {
```

Comment:
@vecerek You will need to handle this case in https://github.com/ManageIQ/manageiq-ui-classic/pull/642 :)

(The id=LIST is needed by backend, but doesn't have to imply miq_grid_checks in general, just for the buttons touched here..)